src/openpgp.js

```diff
@@ -99,7 +99,7 @@ export function destroyWorker() {
 export function generateKey({ userIds=[], passphrase, numBits=2048, unlocked=false } = {}) {
   const options = formatUserIds({ userIds, passphrase, numBits, unlocked });
 
-  if (!util.getWebCryptoAll() && asyncProxy) { // use web worker if web crypto apis are not supported
+  if (asyncProxy) { // use web worker when available
```

Comment:
So this will cause [IE and Edge](https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/13904769-support-web-crypto-api-inside-a-web-worker) to use JS instead of WebCrypto.

I am actually not sure which browsers even support WebCrypto inside a worker thread. See [here](https://lists.w3.org/Archives/Public/public-webcrypto-comments/2013Jul/0012.html).

Have you done benchmarks to validate that this doesn't introduce performance regression?